src/pages/ConfigurePatch.tsx

```diff
@@ -10,17 +10,31 @@ import { ConfigurePatchCore } from "pages/configurePatch/ConfigurePatchCore";
 import { PatchAndTaskFullPageLoad } from "components/Loading/PatchAndTaskFullPageLoad";
 import { PageWrapper } from "components/styles";
 import { usePageTitle } from "hooks";
+import { getVersionRoute } from "constants/routes";
 
 export const ConfigurePatch: React.FC = () => {
   const { id } = useParams<{ id: string }>();
+
   const { data, loading, error } = useQuery<
     ConfigurePatchQuery,
     ConfigurePatchQueryVariables
   >(GET_PATCH_CONFIGURE, {
     variables: { id },
   });
+
   usePageTitle(`Configure Patch`);
 
+  const router = useHistory();
+
+  // redirect to version page if patch is on commit queue
+  useEffect(() => {
+    const commitQueuePosition = data?.patch?.commitQueuePosition ?? null;
```

Comment:
I believe this check fails if the patch has already started. I would check if data.patch.alias = "__commit_queue" to determine if it's a commit queue patch. Same with the other spot

Reply:
great suggestion, thanks!